Add ServiceManifestImport tag

p1p3 opened the pull request:
Clarify that the Policy label is a child element of the ServiceManifestImport tag.

azurecla commented:
Hi __@pshelton-skype__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

brandwe commented:
Actually, can I ask you to fix this? Agree with https:// is fine but you are linking to the actual repo instead of the sample. Sample should be: 

`https://github.com/Azure-Samples/active-directory-android.git` correct?

BryanLa commented:
Hi @pshelton-skype - your PR correction is for the step on cloning the ["Integrating an android app with Azure AD" sample repo](https://github.com/AzureADSamples/NativeClient-Android), ie:

`$ git clone git@github.com:AzureADSamples/NativeClient-Android.git`

The article is actually correct, as the "org" segment of the URL must be **AzureADSamples** (not AzureAD). 

Looking at your error messages though, it looks like they are specific to cloning the [SDK source](https://github.com/AzureAD/azure-activedirectory-library-for-android), which *is* in the Azure AD org. I believe the problem is that you were attempting to clone the SDK source from a URL that had AzureADSamples as the org instead of AzureAD:

`git clone git@github.com:AzureADSamples/azure-activedirectory-library-for-android.git`

so you just need to modify, removing the "Samples" portion, ie: 

`git clone git@github.com:AzureAD/azure-activedirectory-library-for-android.git`

Let me know if that's the case and we can decide if the PR needs to be close, or we're misunderstanding each other. Also, per @brandwe's comment, we have migrated the samples to an entirely new org as well (GitHub.com/azure-samples); for now the old URLs will redirect, but we will be revising the articles over time to indicate this as well. Also note that you can use the [HTTPS form of the repo remote](https://help.github.com/articles/which-remote-url-should-i-use/#cloning-with-https-recommended) for cloning as well, if you prefer.

Thanks!

BryanLa commented:
Closing for now, but we can reopen later after you have a chance to review.

p1p3 committed:
Add ServiceManifestImport tag 

Clarify that the Policy label is a child element of the ServiceManifestImport tag.

GitHubber17 committed:
Merge branch 'master' into patch-2
